assets/css/presenter.css

```diff
@@ -0,0 +1,65 @@
+.presenter {
```

Comment:
Yes I know it's a CSS file but really for developing and changing styles only having to reload a css instead of the whole app with the js madness is what I do.